src/Sylius/Behat/Context/Ui/Shop/LocaleContext.php

```diff
@@ -48,12 +48,12 @@ public function iShowHomepageWithTheLocale(string $localeCode): void
     }
 
     /**
-     * @Then I should shop using the :locale locale
-     * @Then I should still shop using the :locale locale
+     * @Then I should shop using the :itsLocaleName locale
+     * @Then I should still shop using the :itsLocaleName locale
      */
-    public function iShouldShopUsingTheLocale($locale)
+    public function iShouldShopUsingTheLocale(string $itsLocaleName)
```

Comment:
```suggestion
    public function iShouldShopUsingTheLocale(string $LocaleName): void
```
IMO `localeName` will be enough

Reply:
Doesn't work that way :confused: I think the variable name has to be the same as defined in the transformer.